MandelaK/SendIT-API-V2
# [#162030488] set up project structure

#### What does this PR do?
set up project structure

#### Description of Task to be completed?
Before we can begin developing our application it is important to have a properly set up workspace. From there we can configure the application and begin coding straight away.

#### How should this be manually tested?
This cannot be manually tested because no functionality is implemented. However, you can visually inspect the current repository to see how files are organized.

#### What are the relevant pivotal tracker stories?
#162030488 - https://www.pivotaltracker.com/story/show/162030488
